resources/js/components/PracticalTimeLine.vue

```diff
@@ -18,21 +18,7 @@
                     <div class="col-md-2">
                       <div class="bttn-container">
                         <button class="btn btn-secondary" type="submit">
-                          <i class="fa fa-plus"></i>Add to Lab
```

Comment:
@softdiddy can you clarify why this changes? Not sure your current PR has anything to do with this page.

Reply:
no changes sir

Reply:
But it looks like you deleted the from line 21?

Reply:
No, still there